Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/common/DccJWKVerification.kt

```diff
@@ -31,9 +31,10 @@ class DccJWKVerification @Inject constructor(securityProvider: SecurityProvider)
         securityProvider.setup()
     }
 
+    @Throws(DccTicketingJwtException::class)
     fun verify(jwt: String, jwkSet: Set<DccJWK>) {
         // 1. Check for empty jwkSet
-        if (jwkSet.isEmpty()) throw DccTicketingJwtException(JWT_VER_NO_JWKS)
+        if (jwkSet.isEmpty()) throw DccTicketingJwtException(JWT_VER_EMPTY_JWKS)
```

Comment:
Renamed to match `specs`

Reply:
**1. Check for empty jwkSet:** if `jwkSet` is empty, the operation shall abort early with error code `JWT_VER_NO_JWKS`
I don't see any `JWT_VER_EMPTY_JWKS` error code

Source: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/0f1e3a323fd2d52917ffc71ea256d26278e4e16b/docs/spec/dcc-validation-service.md#verifying-the-signature-of-a-jwt-with-a-set-of-jwks

FYI check this commit: https://github.com/corona-warn-app/cwa-app-tech-spec/pull/110/commits/a24822be47637844354807e1b301513c24021323

Reply:
Well the one that you linked in the ticket it does https://github.com/corona-warn-app/cwa-app-tech-spec/blob/f51d1ed62a2d40503d2f41a8e612da155fdf237b/docs/spec/dcc-validation-service.md#requesting-the-result-token

Reply:
tech specs are updating frequently, better always check with the latest PR from https://github.com/corona-warn-app/cwa-app-tech-spec/pulls